SECURITY.md

```diff
@@ -25,11 +25,12 @@ A major version will be supported as long as it relies on a [non deprecated vers
 
 The following table provides the status of the Otter versions under support and each of their corresponding Angular version:
 
-|  Otter version  |  Angular version  |  Support ends  |
-|:---------------:|:-----------------:|:--------------:|
-|        9        |        16         |   2024-11-08   |
-|        8        |        15         |   2024-05-18   |
-|        7        |        14         |   2023-11-18   | 
+| Otter version | Angular version | Support ends |
+| :-----------: | :-------------: | :----------: |
+|      10       |       17        |      -       |
+|       9       |       16        |  2024-11-08  |
+|       8       |       15        |  2024-05-18  |
+|       7       |       14        |  2023-11-18  |
```

Comment:
do we keep this version since it is no longer supported since 2023 ?

Reply:
I would say yes, removing them does not bring any value.
(I asked myself the same question)